resources/definitions/ultimaker.def.json

```diff
@@ -154,6 +151,56 @@
         },
         "bridge_fan_speed_3": {
              "value": "cool_fan_speed_min"
+        },
+        "bridge_fan_speed_3": {
+             "value": "cool_fan_speed_min"
+        },
```

Comment:
the bridge_fan_speed_3 is now defined twice, seems to be a mistake

Reply:
Indeed , this seems to be a merge error. I will correct it.